hermes-console/static/partials/modal/editGroup.html

```diff
@@ -4,6 +4,9 @@ <h3 class="modal-title" ng-show="operation === 'ADD'"><small>Create group:</smal
         <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit group:</small> {{group.groupName}}</h3>
     </div>
     <div class="modal-body">
+        <div ng-show="!admin && operation == 'ADD'" class="alert alert-warning">When creating a group please make sure that it complies with general
+            group standards and naming conventions. In addition, ensure that there does not already exist a group, that would fit your needs</div>
```

Comment:
Maybe that is a little bit more readable? :)
```suggestion
            group standards and naming conventions. Confirm if a group that meets your requirements does not already exist.</div>
```